code/modules/cargo/packs.dm

```diff
@@ -1507,6 +1507,28 @@
 					/obj/item/stack/tile/carpet/black/fifty)
 	crate_name = "premium carpet crate"
 
+/datum/supply_pack/service/carpet_exotic
+	name = "Exotic Carpet Crate"
+	desc = "Exotic carpets straight from Russia, for all your decorating needs. Contains 100 tiles each of 8 different flooring patterns.
```

Comment:
you forgot the closing quotation mark for the string

Reply:
god bless

Reply:
Space Russia*